WalletWasabi.Tests/IntegrationTests/TorTests.cs

```diff
@@ -143,6 +144,26 @@ public async Task CanDoBasicPostHttpsRequestAsync()
 		Assert.Equal("https://postman-echo.com/post", GetJsonNode(responseNode, "url").GetValue<string>());
 	}
 
+	/// <summary>
+	/// Downloads a binary file and verifies its checksum to prove it's correctly downloaded.
+	/// </summary>
+	[Fact]
+	public async Task DownloadBinaryFileAsync()
+	{
+		using CancellationTokenSource ctsTimeout = new(TimeSpan.FromMinutes(2));
+
+		TorHttpClient client = MakeTorHttpClient(new Uri("https://bitcoin.org"));
+
+		using HttpRequestMessage request = new(HttpMethod.Get, "https://bitcoin.org/bitcoin.pdf");
```

Comment:
Why not use https://wasabiwallet.io/bitcoin-whitepaper.pdf instead? It's something we have control over from the server side too.

Reply:
Advantage is that we control that file exists there, not some other party. So, if tests starts to fail, it's either Wasabi codebase or stupid wasabiwallet.io admins. :)